docs/cow-protocol/reference/core/auctions/schema.md

```diff
@@ -15,441 +18,192 @@ To avoid precision loss, some numerical literals are encoded as strings, referre
+### `liquidity`
 
-```json
-"52": {
-    "kind": "WeightedProduct",
-    "reserves": {
-        "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599": {
-            "balance": "100000",
-            "weight": "0.4"
-        },
-        "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48": {
-            "balance": "31791255",
-            "weight": "0.2"
-        },
-        "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2": {
-            "balance": "17015250000000000",
-            "weight": "0.4"
-        }
-    },
-    "fee": "0.0055",
-    "cost": {
-        "amount": "14827198783080000",
-        "token": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
-    }
-}
-```
+This key is a list of all the publicly available AMMs/limit orders that are made available for use by the default implementation of the Driver, available to the solver engines that request such liquidity. Internally, this is also known as _baseline_ liquidity. Each entry describes the current state of an AMM or an existing public limit order. Currently, there are 5 different types of liquidity provided by the Driver:
 
-#### Stable Pools
+- constant product pools, i.e., Uniswap v2 type pools on 2 tokens.
+- weighted Product pools, i.e., Balancer type weighted product pools on N tokens.
+- stable pools, i.e., Curve type stable pools on N tokens.
+- concentrated liquidity pools, i.e., Uniswap v3 type concentrated liquidity pools on 2 tokens.
+- foreign limit orders, i.e., external 0x type limit orders on 2 tokens.
 
-A Stable pool corresponds to [Curve/Balancer stable pools](https://dev.balancer.fi/resources/pool-interfacing/stable-pool), and has the following differences to the Constant Product pool described above,
+More information about the exact descriptions of these pools can be found [here](https://docs.cow.fi/cow-protocol/reference/apis/solver).
 
-- `"kind"`: the type is set to "Stable".
-- `"amplification_parameter"`: See [amplification coefficient](https://curve.fi/files/stableswap-paper.pdf).
-- `"scaling_rates"`: See [getScalingFactors](https://github.com/balancer-labs/balancer-v2-monorepo/blob/80b0e1b129d575c313f59800ec7e19237a43c087/pkg/pool-utils/contracts/BasePool.sol#L523-L525).
+### `effectiveGasPrice`
 
-An example of such an AMM between [DAI](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f), [USDC](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) and [USDT](https://etherscan.io/token/0xdac17f958d2ee523a2206206994597c13d831ec7) is given below.
+This key is a single entry that is a stringified integer describing an estimate of the effective gas price for the winning solution, denominated in wei.
```

Comment:
Maybe it was better before. As in
```suggestion
This key is a single entry that is a stringified integer describing the current gas price in wei.
```

What I wanted to get at was that solvers interpreted the number as an estimate of the gas_price and that only general variations in gas price would lead to a different value for the settlement. But this is wrong, if I understand correctly. We do in some cases add 10% or something like that. This additional tip seems to be the main source of deviation and tripped up solvers in the past.

Reply:
But i believe this is not just the base fee but it tries to approximate the effective gas price including the fee. Maybe I am wrong here. Let me try to check the code and get back to you.

Reply:
https://github.com/cowprotocol/services/blob/06e9c4ed58da488973299c14164c452c59505d72/crates/driver/src/domain/eth/gas.rs#L66

This seems to suggest tip is already added here.

Reply:
At some point is was base fee + "small tip", where the small tip was for inclusion of fully internalized transactions. This is then a good estimate for the case where there are no interactions, just transfers to users.
When there are interactions, we added an additional tip, so that block builders include our transaction faster or, rather, we pay them to not extract MEV. Not sure if that changed at some point. @sunce86 

Also not super important. I just remembered that solvers did have problems understanding the meaning which lead to "unintentional" overbidding.